app/views/courses/_course_table_row.html.erb

```diff
@@ -25,7 +32,11 @@
   <% if @recruitment_cycle.current_and_open? %>
     <td class="govuk-table__cell" data-qa="courses-table__vacancies">
       <% if course.is_running? || course.is_withdrawn? %>
-        <%= course.vacancies %>
+        <% if current_page?(provider_recruitment_cycle_courses_path(@provider.provider_code, course.recruitment_cycle_year)) %>
+          <%= course.vacancies %>
+        <% else %>
+          <%= course.has_vacancies? ? "Yes" : "No" %>
```

Comment:
I found out today that this is a decorator that already returns "Yes" or "No". Be worth checking..

Reply:
`vacancies` returns "Yes (Edit)" with a link or "No". 
I thought it may be easiest to just do it here. Alternatively, I could create a decorator method for it (but I wasn't sure about the name vacancies_for_courses_as_accredited_body / vacancies_without_edit_link)

Reply:
Ah yes, good spot! This is fine